scripts/apps/publish/directives/SubscribersDirective.ts

```diff
@@ -262,13 +264,15 @@ export function SubscribersDirective(
                 $scope.newDestination = null;
             };
 
-            $scope.$watchCollection('subscriber', (newValue, oldValue) => {
-                if (newValue && oldValue && newValue !== oldValue) {
+            $scope.$watch('subscriber', (newValue, oldValue) => {
+                if (newValue && oldValue && !_.isEqual(newValue, oldValue)) {
```

Comment:
Why isn't it enough to handle this here? Why do you need `enableSave` method?

Reply:
Tokens are stored in different collection and making any changes to them does not affect `subscriber`. So actually, there is no need to enable the save button on token generation. But Karel said that we must enable it from a users point of view.

Reply:
Can you watch both collections and enable save button when any of them changes? I assume that would allow to get rid of `enableSave` method. Or at least only keep it in one place only and not use it in the UI templates.

Reply:
Both collections can't be watched together as one of them is the part of `controller as` syntax. But I have removed the `enableSave` method.